modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -18,18 +19,140 @@ As an Operator author, your Operator must meet additional requirements to run pr
 // TODO: Include more info w/ better steps on how to do this:
 //* You must understand the {product-title} proxy configuration.
 
-For the CSV requirements, you can make the following changes as the Operator author.
+For the CSV requirements, you can make the following changes as the Operator author. The following procedure uses the Memcached Operator created in the Operator SDK tutorials for Go, Ansible, and Helm.
 
 .Prerequisites
 
 * An Operator project with a CSV.
 
 .Procedure
 
-. Use SHA references to related images in two places in the CSV for your Operator:
+. Set an environment variable, such as `RELATED_IMAGE_MEMCACHED`, for any additional images used by the Operator in the `config/manager/manager.yaml` file:
++
+.Example `config/manager/manager.yaml`
+[%collapsible]
+====
+[source,yaml]
+----
+...
+spec:
+  ...
+    spec:
+      ...
+      containers:
+      - command:
+        - /manager
+        ...
+        env:
+        - name: RELATED_IMAGE_MEMCACHED
+          value: "docker.io/memcached:1.4.36-alpine"
+----
+====
+
+. Pull the related images with digests by replacing hard-coded image references with environment variables in the following files:
```

Comment:
Here I got a little confused bc after I see in the following files I kind of expected a more straightforward list. It might be better to either cut number 2 off as a sentence "Pull the related images with digests by replacing hard-coded image references with environment variables" and then have the files listed in the bullets below like you have them or you could list out the files somehow.